### PR TITLE
Update workshop-resources file

### LINKED ISSUE #122: Break up workshop-resources.md
_Broken out of https://github.com/AlexsLemonade/training-specific-template/issues/109_

In #119 the file formerly known as `resources-for-consultation-sessions.md` will have been renamed to `workshop-resources.md`.
In #120, new files will have been created in `additional-resources` that contain much of the content from the original single resources file, broken up into bulk vs. single-cell scopes.

For this issue, we'll take the next steps:
- Move any relevant screenshots for those contents out of `workshop/screenshots` and into `additional-resources/screenshots`.
- Update `workshop-resources.md` itself to include just general resources, including information about how consultation sessions work. This will also include adding links to single-cell and bulk resources, _and_ link to the R resources too while we're here.

### PR DESCRIPTION
**Pull request by sjspielman**
~**⚠️ Stacked on #128**~
Closes #122 

This PR updates the `workshop/workshop-resources.md` file.

I deleted content that was moved out into the new bulk & single-cell resources files created in `additional-resources` as part of #127, and created sections here to link those files (as well as the R resources file) from here. I also cleaned up some text in this markdown to make it more focused on resources and less on consultations, added all the cheatsheets, and fixed a relative link that became deprecated.

### CODE REVIEW

**Comment by sjspielman** [comment]
Requesting review now, but _not_ a priority this sprint!

**Comment by sjspielman** [comment]
Noting I merged in the GHA to copy completed notebooks, but should not cause "clutter" for review here since only one file! So, no need to review `.github/workflows/copy-completed-notebooks.yml` here (unless you see something that we missed!)

File: workshop/workshop-resources.md [hunk 1]
```diff
@@ -38,199 +22,35 @@ On this page, we've assembled some resources you may find helpful during these s
...
-
-Ensembl GRCm38 (mm10) v95
-
-| File description | File use | File path |
-|------------------|----------|-----------|
-| Mouse Salmon index `-k 23` | Salmon index for use with `salmon quant`; appropriate for reads shorter than 75bp or for increased sensitivity with `--validateMappings` ([docs](https://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-mapping-based-mode)) | `~/shared-data/reference/refgenie/mm10_cdna/salmon_index/short` |
-| Mouse Salmon index `-k 31` | Salmon index for use with `salmon quant`; appropriate for reads 75bp or longer ([docs](https://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-mapping-based-mode)) | `~/shared-data/reference/refgenie/mm10_cdna/salmon_index/long` |
-
-### _Danio rerio_
+## Resources for getting started with R programming
```
Comment (jashapiro) [inline workshop/workshop-resources.md:44]: The "Resources for" is a bit redundant in these three headers. Maybe nest these something like:

```suggestion
## Topic-specific resources 

### Getting started with R programming
```

and similar for the below sections?

Comment (sjspielman) [reply workshop/workshop-resources.md:44]: can do!

**Pull request merged**